src/baseline.es

```diff
@@ -17,9 +17,9 @@ module.exports = Class.extend( {
 		} );
 	},
 
-	merge : function( data, options ) {
-		var options = _.extend( {}, { empty : false }, options );
-		var _this = this;
+	merge( data, defaultOptions ) {
+		const options = _.extend( {}, { empty : false }, defaultOptions );
+		const _this = this;
 
 		if( data ) {
 			_.each( data, function( thisServiceData, thisServiceIdent ) {
```

Comment:
we should use native javascript functions (as opposed to underscore functions) when possible, i.e. `.forEach`

Reply:
Yes! also switched _.contains for array.inlcudes(). Thanks!